runtests.py

```diff
@@ -45,22 +44,45 @@
                 },
             },
         ],
-        ROOT_URLCONF='tos.tests.test_urls',
-        LOGIN_URL='/login/',
-        SITE_ID='1',
-        CACHES = {
+        'ROOT_URLCONF': 'tos.tests.test_urls',
+        'LOGIN_URL': '/login/',
+        'SITE_ID': '1',
+        'CACHES': {
             'default': {
                 'BACKEND': 'django.core.cache.backends.dummy.DummyCache',
             },
             'tos': {
                 'BACKEND': 'django.core.cache.backends.locmem.LocMemCache',
             }
         },
-        TOS_CACHE_NAME = 'tos' 
-    )
+        'TOS_CACHE_NAME': 'tos' 
+    }
+
+    if django.VERSION >= (1, 10, 0):
```

Comment:
Based on the [support version](https://www.djangoproject.com/download/#supported-versions), as of April 2020 only **2.2 LTS and newer** are supported.